packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -15,7 +15,7 @@
 
   .layout {
     display: flex;
-    font-family: 'Roboto', sans-serif;
+    font-family: var(--terra-consumer-font-family, 'Roboto', sans-serif);
```

Comment:
terra base actually has this property. We should remove it from layout and just use there:

https://github.com/cerner/terra-core/blob/master/packages/terra-base/src/Base.scss#L8

Reply:
@mhemesath, should this jira be used to also integrate with terra base? technically its listed as a dependency, we aren't using it though.
Should that be something done in layout? or the consuming repo (like healthelastic-react)?

cc @bjankord 
where is the terra core layout code? i can't seem to find it. curious what they are doing.